local/src/core/http.cc

```diff
@@ -417,7 +435,8 @@ HttpHeader::verify_headers(swoc::TextView transaction_key, HttpFields const &rul
         }
       } else {
         if (!rule_check
-                 ->test(transaction_key, field_iter->first, swoc::TextView(field_iter->second))) {
+                 ->test(transaction_key, field_iter->first, swoc::TextView(field_iter->second)))
+        {
```

Comment:
huh...I know you didn't do this, but it seems strange that this `->test` is on a separate line. Can you join it with the previous like and re-run clang-format to see whether it breaks it up again? I wonder whether a space was added here accidentally which then broke it on a separate line.

Reply:
Yes vscode(clang-format in the c/c++ extension) did that for me :)

I tried to revert as you suggested and run `clang-format`,  but it breaks it up again. Actually the `->test` was there on a separate line even before my change. 

The only difference is my change put the curly braces on it's own line rather than on the same line as previously. I think this has to do with the `clang-format` version, even when specified to use the same `.clang-format` style file, the output can be different.

Reply:
Thanks for checking.